R/post_overline.R

```diff
@@ -35,9 +35,9 @@ post_overline = function(
 
   overlapping_aggregated = overlapping_joined |>
     sf::st_drop_geometry() |>
-    dplyr::mutate(across(matches("bicycle"), function(x) x * length_y)) |>
+    dplyr::mutate(across(matches("bicycle|gradient|quietness"), function(x) x * length_y)) |>
     dplyr::group_by(osm_id) |>
-    dplyr::summarise(across(matches("bicycle"), \(x) sum(x, na.rm = TRUE)), .groups = "drop")
+    dplyr::summarise(across(matches("bicycle|gradient|quietness"), \(x) sum(x, na.rm = TRUE)), .groups = "drop")
```

Comment:
This is not quite right: quietness and gradient should not be summed. You can just take the most highest value for both.